spec/init_handler_spec.cr

```diff
@@ -6,7 +6,7 @@ describe "Kemal::InitHandler" do
     io = IO::Memory.new
     response = HTTP::Server::Response.new(io)
     context = HTTP::Server::Context.new(request, response)
-    Kemal::InitHandler::INSTANCE.next = ->(context : HTTP::Server::Context) {}
+    Kemal::InitHandler::INSTANCE.next = ->(_context : HTTP::Server::Context) {}
```

Comment:
In terms of Crystal semantic, there is no difference between this names. But I think it is better to explicitly show the argument is not used prepending the underscore. So this is more a change of a style rather than a fix of a code smell. If you disagree with it, i will revert.